matrix/instrumentations.yaml

```diff
@@ -0,0 +1,303 @@
+- keys:
+    - ASPNET
+  library: ASP.NET Framework (.NET Framework)
+  version: See :ref:`dotnet-otel-versions`
+  comment: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
+  stability: beta
+  supportabilityLevel: community
+  dependencies:
+    - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet
+      version: 1.0.0-rc9.9
+      stability: beta
+    - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule
+      version: 1.0.0-rc9.9
+      stability: beta
+  toDoComment: version with ref link
```

Comment:
?

Reply:
TODO commenta are related to the content of the data, if it is present we need to find a solution how to handle this cases.

In this example, reflink (documentation) is used. So our metadata will be tight to particular version of documentation.